tests/longevity/results/1.1.0/1.1.0.md

```diff
@@ -145,47 +147,61 @@ No such responses.
 
 ![memory.png](memory.png)
 
-Memory usage dropped twice which appears to correspond with the restarts seen above relating to leader election.
-Interestingly, before the first restart and after the second restart, memory usage sat at about 8.5MiB, but for the
-majority of the test run, memory usage was about 9.5-10MiB. The previous release test run also had memory usage at
-about 9-10MiB, but more stable usage across the duration of the test. However, there were no restarts observed in the
-v1.0.0 test run. I don't think there is anything to investigate here.
+Memory usage dropped once which appears to correspond with a restart event seen above relating to leader election.
+There did seem to be a creeping up of memory before this restart, and then memory usage remained consistent for the rest
+of the test. Memory usage was slightly higher than what was observed in the v1.0.0 longevity test, but unless we see
+this trending upwards on future longevity tests, I don't think there is anything to investigate here.
 
 ### Containers CPU
 
 ![cpu.png](cpu.png)
 
-No unexpected spikes or drops.
+One drop in NGINX CPU that corresponds with a Pod termination event.
 
 ### NGINX metrics
 
-In this test, NGINX metrics were not correctly exported so no dashboards are available for these.
+![stub-status.png](stub-status.png)
+
+The drop of _requests_ on Dec 16 corresponds to a Pod termination event. There are no correlated errors in NGINX logs.
```

Comment:
if the pod terminates because of the leader election lost, not just NGF container restart, I think it is a problem - cause this means transient problems in the Kubernetes API can affect the traffic going through NGINXes, because pods are restarted.

Reply:
The pod termination events happened about 10 minutes after the leader election loss restarts. The leader election loss didn't cause the termination. It feels like external factors.